srcpkgs/virtualbox-ose/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'virtualbox-ose'
 pkgname=virtualbox-ose
 version=6.1.14
-revision=1
+revision=2
 wrksrc="VirtualBox-${version%*a}"
 short_desc="General-purpose full virtualizer for x86 hardware"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only, CDDL-1.0"
 homepage="https://www.virtualbox.org"
 changelog="https://www.virtualbox.org/wiki/Changelog"
-distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-$version.tar.bz2"
-checksum=91fa05bcfce36316ca93e3927c9550ea66286fff4c5bec900b753fca278ce1a0
+distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-6.1.14a.tar.bz2"
```

Comment:
Stay with $version and just append the `a` to the version string. And don't bump the revision.

Reply:
distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-$versiona.tar.bz2"

Reply:
`distfiles="http://download.virtualbox.org/virtualbox/${version%*a}/VirtualBox-${version}a.tar.bz2"` to make it clearer, imo